mtp_api/apps/mtp_auth/admin.py

```diff
@@ -34,6 +37,17 @@ class RoleAdmin(ModelAdmin):
     def all_groups(self, instance):
         return ', '.join(sorted(group.name for group in instance.groups))
 
+@admin.register(JobInformation)
+class JobInformationAdmin(ModelAdmin):
+    list_display = ('title', 'prison_estate', 'tasks', 'fullname', 'email', 'user_id')
+    list_filter = ('title', 'prison_estate')
```

Comment:
i'm not sure how well `prison_estate` will work here since it's a free text field as far as this app is concerned

Reply:
Ah - in the form we have three options so that should control options initially. There's no other option on the question so hopefully should be OK initially.

Reply:
fair enough